doc/release-notes-27217.md

```diff
@@ -0,0 +1,6 @@
+Wallet
+------
+
+* Purposes strings are restricted to the currently known values of "send", "receive", and "refund".
```

Comment:
In commit "doc: Release note for purpose string restriction" (0d18142f595b43070bd07b608d43c9ed9b37cdcc)

Maybe s/restricted/now restricted/ to be clear this is new behavior not background information.

Reply:
Done